### PR TITLE
Sens fixes

### PR DESCRIPTION
**Pull request by vuillaut**
Some changes to make sensitivity run to the end.
The results are still completely wrong though.
I think you'll need to merge master in your branch before having this PR synched

### CODE REVIEW

**Comment by rlopezcoto** [comment]
I did, but there are still have some conflicts that need to be solved...

File: lstchain/mc/sensitivity.py [hunk 1]
```diff
@@ -402,11 +410,14 @@ def find_best_cuts_sensitivity(simtelfile_gammas, simtelfile_protons,
 
     # Weight events and count number of events per bin:
     for i in range(0, n_bins_energy):  # binning in energy
+        total_rate_proton_ebin = np.sum(rate_weighted_p[(e_reco_p < energy[i + 1]) & (e_reco_p > energy[i])])
```
Comment (rlopezcoto) [inline lstchain/mc/sensitivity.py:412]: why is this repeated?

File: lstchain/mc/sensitivity.py [hunk 2]
```diff
@@ -314,38 +325,36 @@ def find_best_cuts_sensitivity(simtelfile_gammas, simtelfile_protons,
...
+    emax_sensitivity = 10 ** 5 * u.GeV  # mc_par_g['emax']
 
     energy = np.logspace(np.log10(emin_sensitivity.to_value()),
-                np.log10(emax_sensitivity.to_value()), n_bins_energy + 1) * u.GeV
+                         np.log10(emax_sensitivity.to_value()), n_bins_energy + 1) * u.GeV
 
     gammaness_bins, theta2_bins = bin_definition(n_bins_gammaness, n_bins_theta2)
 
-    # Extract spectral parameters
+    # Extract spectral parameterss
```
Comment (rlopezcoto) [inline lstchain/mc/sensitivity.py]: typo

File: lstchain/mc/sensitivity.py [hunk 3]
```diff
@@ -69,14 +71,17 @@ def process_mc(dl1_file, dl2_file, mc_type):
     mc_par:    `dict` with simulated parameters
 
     """
+    from ..io.io import dl2_params_lstcam_key
```
Comment (rlopezcoto) [inline lstchain/mc/sensitivity.py]: already included following the comments on the main PR

**Review by rlopezcoto — COMMENTED** [review]
Thanks for the help @vuillaut! 
So from these changes I think we are still not correcting the error from the lstchain tests, right?

File: lstchain/mc/sensitivity.py (hunk 1 above)
Comment (vuillaut) [reply lstchain/mc/sensitivity.py:412]: a mistake in resolving conflicts, thanks

**Comment by vuillaut** [comment]
> Thanks for the help @vuillaut!
> So from these changes I think we are still not correcting the error from the lstchain tests, right?

Maybe not, I have checked that it could run with standard gamma/protons files but the gamma test file might pose other problems... 
However, the output is still not physical :-/

**Review by rlopezcoto — APPROVED** [review]

**Pull request merged**